CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ Increment the:
   [#2631](https://github.com/open-telemetry/opentelemetry-cpp/pull/2631)
 * [SDK] DefaultLogHandler to print errors to std::cerr, add LogLevel::None
   [#2622](https://github.com/open-telemetry/opentelemetry-cpp/pull/2622)
+* [SDK] Update ExemplarFilter and ExemplarReservoir for spec
```

Comment:
This CHANGELOG entry should go in the unreleased section.

Reply:
Thanks for catching.